gradle.properties

```diff
@@ -1,4 +1,4 @@
-release_version=3.8.0
+release_version=4.0.0
```

Comment:
We should update only the minor version because these commits contain only extended changes

Reply:
And also target branch should be **th2-2150-books-pages**?
I thought that we increase version in all components because we use cradle 4.

Reply:
This library knows nothing about Cradle

Reply:
Thanks, I see. I'll fix it.

Reply:
done